tests/core/pyspec/eth2spec/test/deneb/sanity/test_blocks.py

```diff
@@ -62,15 +65,18 @@ def test_invalid_incorrect_blob_tx_type(spec, state):
     opaque_tx = b'\x04' + opaque_tx[1:]  # incorrect tx type
     block.body.execution_payload.transactions = [opaque_tx]
     block.body.execution_payload.block_hash = compute_el_block_hash(spec, block.body.execution_payload)
-    signed_block = state_transition_and_sign_block(spec, state, block, expect_fail=True)
+    signed_block = state_transition_and_sign_block(spec, state, block)
 
     yield 'blocks', [signed_block]
-    yield 'post', None
+    yield 'post', state
 
 
 @with_deneb_and_later
 @spec_state_test
-def test_invalid_incorrect_transaction_length_1_byte(spec, state):
+def test_incorrect_transaction_length_1_byte(spec, state):
```

Comment:
Should we do a version of this that has executionengine return invalid?

Reply:
Essentially, the `execution_valid` result is fully up to what we set in the testing script. So I think only the "the input is correct, but `execution_valid` returns False" is valuable here. I added test case `test_invalid_correct_input__execution_invalid` for it.